# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/MediaTrackGraph.h
# Commit: a6d656aa59c0
# Full Hash: a6d656aa59c097a7867e905c36c8bccfbef548ff
# Author: Chun-Min Chang <chun.m.chang@gmail.com>
# Date: 2022-01-31 21:28:43
# Regressor Bug: 1742655
# File Overlap Count: 2
# Description:
#   Bug 1752589 - Rename AsAudioInputTrack() to AsAudioProcessingTrack() r=padenot
#   
#   The `AudioInputTrack` is renamed to `AudioProcessingTrack` in D134629,
#   in Bug 1742655, so the `AsAudioInputTrack` should be renamed to
#   `AsAudioProcessingTrack` as well.
# ==============================================================================

diff -r 137918f951cc -r a6d656aa59c0 dom/media/MediaTrackGraph.h
--- a/dom/media/MediaTrackGraph.h	Mon Jan 31 17:03:55 2022 +0000
+++ b/dom/media/MediaTrackGraph.h	Mon Jan 31 17:13:49 2022 +0000
@@ -352,7 +352,7 @@
   friend class MediaInputPort;
   friend class AudioNodeExternalInputTrack;
 
-  virtual AudioProcessingTrack* AsAudioInputTrack() { return nullptr; }
+  virtual AudioProcessingTrack* AsAudioProcessingTrack() { return nullptr; }
   virtual SourceMediaTrack* AsSourceTrack() { return nullptr; }
   virtual ProcessedMediaTrack* AsProcessedTrack() { return nullptr; }
   virtual AudioNodeTrack* AsAudioNodeTrack() { return nullptr; }